core/src/main/java/lucee/runtime/coder/HexCoder.java

```diff
@@ -89,9 +89,9 @@ else if (c >= 'a' && c <= 'f') {
 			retorno = (((byte) c) - 87);
 		}
 		else {
-			throw new CoderException("invalid hexadecimal String");
+			throw new CoderException("invalid hexadecimal String for, [ " + hexa + " ]. Allowed characters are [0-9], [a-f], [A-F]");
```

Comment:
`c` not `hexa`

Reply:
I should have stayed retired! Fixed. Thanks.

Reply:
building locally is really easy

https://docs.lucee.org/guides/working-with-source/build-from-source.html